client/src/components/GameArea.js

```diff
@@ -189,4 +191,4 @@ GameArea.propTypes = {
   cards: PropTypes.array.isRequired,
 };
 
-export default GameArea;
+export default withAuthentication(GameArea);
```

Comment:
I'm curious why we wrap in here rather than when another component imports the GameArea.

Reply:
No real reason, it'd probably be better to wrap this where we import it actually. Will push a new commit